app/api/users.py

```diff
@@ -183,6 +183,9 @@ def before_update_object(self, user, data, view_kwargs):
         if data.get('email') and data['email'] != user.email:
             view_kwargs['email_changed'] = user.email
 
+        if data.get('is_admin') != user.is_admin:
+            data['is_admin'] = not user.is_admin
```

Comment:
What is this even doing ??
This is doing absolutely nothing IMO. 
Let `data.get('is_admin')` = A and `user.is_admin` = B
1. A = true, B = true - The if statement isn't true so nothing happens
2. A = false B = True - So the if statement is correct. Now you set A = not B and hence A = not True = False. But A is already false. 
3. A = true, B = False - If statement is true. Now you set A = not B and hence A = not False = True, but A is already true.
4. A = False, B = false - If statement isn't true and hence it won't run. 

How does this have any impact on the values ???